src/ebookmaker/writers/__init__.py

```diff
@@ -61,6 +62,55 @@ def write_with_crlf(filename, bytes_):
         with open(filename, 'wb') as fp:
             fp.write(bytes_)
 
+    @staticmethod
+    def writec_with_crlf(filename, bytes_):
```

Comment:
you are substituting a method that takes bytes and saves bytes with a method that takes bytes and saves a string.
instead, write a method that operates on strings and apply it separately from the cross-platform method that normalizes line endings. Maybe call your method nice_wrap(string). As it's written, it may produce wrong line endings on one or more platforms.

I don't think this method is even necessary - you should be able to tweak the generating code to do what you want.